frontend/components/Gallery/index.jsx

```diff
@@ -0,0 +1,30 @@
+import "@fancyapps/ui";
+import styles from "./gallery.module.css";
+import {useGetPhotoGalleryQuery} from "../../store/api/categories";
+import Image from 'next/image';
+
+export default function Gallery() {
+    const {data: photoList} = useGetPhotoGalleryQuery();
+
+    return (
+        <div className={styles.container}>
+            {photoList.map(photo => (
+                <div key={photo.id} className={styles.wrapper}>
+                    <a
+                        className={styles.fancybox_wrap}
+                        data-fancybox=''
+                        data-src={`${process.env.NEXT_PUBLIC_URL}/${photo.photo_url}`}
```

Comment:
Why do we need this prefix `process.env.NEXT_PUBLIC_URL` here? Backend not returning the full url to the photo?

Reply:
Yes, the backend does not return the full URL, so this prefix is substituted here.